docs/cardinal/game/message.mdx

```diff
@@ -3,7 +3,7 @@ title: 'Message'
 description: 'How to define and register a message'
 ---
 
-Messages are predefined user actions that can be submitted to Cardinal and handled within a system. A message is the primary way of representing possible user actions in the game. After a message is executed within a tick, it returns a `Reply` payload that can be read by the game client.
+Messages are predefined user actions that can be submitted to Cardinal and handled within a system. A message is the primary way of representing possible user actions in the game. After a message is processed within a tick, it returns a `Reply` payload that can be read by the game client.
 
 In Cardinal, transactions are defined using a pair of Go structs representing a `Message` and a `Reply`.
```

Comment:
i realize this is not part of the diff, but maybe we can fix this here?
```suggestion
In Cardinal, messages are defined using a pair of Go structs representing a `Message` and a `Reply`.
```

Reply:
done